src/services/nxtp.tsx

```diff
@@ -75,7 +75,7 @@ export const getTransferQuote = async (
   initiator?: string,
 ): Promise<AuctionResponse | undefined> => {
   // Create txid
-  const transactionId = getRandomBytes32();
+  const transactionId = getRandomBytes32()
```

Comment:
Maybe we use that for our step ids as well?

Reply:
Yes, that would be good.